prow/pod-utils/gcs/upload_test.go

```diff
@@ -17,19 +17,19 @@ limitations under the License.
 package gcs
 
 import (
+	"cloud.google.com/go/storage"
```

Comment:
nit: we want imports ordered by stdlib, third party, then internal -- please revert this import ordering change

Reply:
done, my IDE did that.